### PR TITLE
[no ticket] add prod sd config for underlay

### PR DESCRIPTION
**Pull request by chenchals**
- added dev, prod and test configs for service
- No change to indexer -> we copy the indexed dataset to needed datasets in test and prod

### CODE REVIEW

File: underlay/src/main/resources/config/service/sd/sd020230831_test.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
     "indexData": {
       "projectId": "sd-vumc-tanagra-test",
-      "datasetId": "indexed_sd_20230831_beta"
+      "datasetId": "indexed_sd_20230831"
```
Comment (marikomedlock) [inline underlay/src/main/resources/config/service/sd/sd020230831_test.json:14]: Is this change intended?

**Review by marikomedlock — APPROVED** [review]

File: underlay/src/main/resources/config/service/sd/sd020230831_test.json (hunk 1 above)
Comment (chenchals) [reply underlay/src/main/resources/config/service/sd/sd020230831_test.json:14]: yes-> we copy
